rca/static_src/sass/components/_hero-action-pane.scss

```diff
@@ -1,6 +1,6 @@
 .hero-action-pane {
     $root: &;
-    $hero-action-pane-height-small: 110px;
+    $hero-action-pane-height-small: 190px;
```

Comment:
I explored options where we don't hard code this but think we're going to have to refactor quite a bit if we wanted to.

Reply:
Hey @patrickcuagan I think we should explore this further with RCA.

If we make this change and the content they add to these fields is later reduced, we could end up with the content appearing to high up:

![Screenshot 2024-09-20 at 08 25 50](https://github.com/user-attachments/assets/f158db88-66aa-4e0d-b549-3b9d2c53c0f5)

Reply:
Agreed to proceed with this, but we'll create a separate ticket for refactoring this component in future.